index.html

```diff
@@ -8,24 +8,29 @@
     <title>my-portfolio</title>
 </head>
 <body>
-    <section class="container">
-        <header>
-            <nav>
-                <a href="#">
-                    <img class="my-logo" src="images/my-logo.png" alt="logo">
-                </a>
-                <a href="#">
-                    <img class="menu" src="images/union.png" alt="union">
-                </a>
-            </nav>
-        </header>
+    <section class="heading">
+            <header>
+                <nav class="site-nav">
+                    <a href="#">
+                      <img class="my-logo" src="images/my-logo.png" alt="logo">
+                    </a>
+                    <a href="#">
+                       <img class="menu" src="images/union.png" alt="union">
+                    </a>
+                    <ul>
+                       <li><a href="#">Portolio</a></li>
+                       <li><a href="#">About</a></li>
+                       <li><a href="#">Contact</a></li>
+                    </ul>
+                </nav>
+            </header>
         <section class="main-section">
             <br>
-            <h1 class="main-text">I’m Godanah
+            <h1 class="main-text">I’m Godanah <br>
                 Glad to see you!</h1>
             <p class="content">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
             <br>
-            <p>LET'S CONNECT</p>
+            <p class="p-page1">LET'S CONNECT</p>
             <br>
```

Comment:
- [ ] Kindly, let us readjust the padding-top for our heading to get the same we have on the figma
![port43](https://user-images.githubusercontent.com/55635977/130281642-61ee0006-b9b8-42ca-90e7-9bb26dfea771.png)

Reply:
Fixed. Thanks for the review